ablator/main/mp.py

```diff
@@ -319,6 +442,26 @@ def launch(
         auxilary_modules: list[tys.ModuleType] | None = None,
         ray_head_address: str | None = "auto",
     ):
+        """
+        Set up and launch the parallel training and tuning process. This includes:
+        prepare ray cluster for running optuna trials to tune hyperparameters; if available,
+        synchronize GCP buckets to working directory defined in runtime configuration;
+        initialize optuna trials and add them to optuna storage and experiment state
```

Comment:
should we add some function calls or class names to make it more clear,like experiment state is for storing and managing optuna samples,and sync_up is for synchronization, so users can refer to these files for more details